### DISCUSSION TITLE
Keep getting "Your Copilot experience is not fully configured, complete your setup" in Visual Studio 2022.

### DISCUSSION THREAD

**Discussion by ChristianSHansen** [community/community#19556] [Copilot Conversations] [answered]
I keep getting a pop-up message stating that my "Copilot experience is not fully configured, complete your setup." I have set it up properly in Github, uninstalled the Copilot extension and re-installed it. However, I keep getting the popup multiple times when editing a file in Visual Studio 2022.

![image](https://user-images.githubusercontent.com/102748231/175959835-3b1c9061-9b1f-4894-a5d1-4f27e79bec88.png)

Help!!!

**Comment by ChristianSHansen** [comment, marked as answer]
I figured out my problem. I had signed up to Copilot under a different GitHub account and I needed to delete this directory: "C:\Users\%USERNAME%\AppData\Local\github-copilot" so it could recreate the settings for the correct account.

**Comment by dstsrc** [comment]
I have the same problem on mac os, jetbrains goland 2022.2.
what should i remove?

<img width="447" alt="Снимок экрана 2022-08-09 в 14 05 35" src="https://user-images.githubusercontent.com/65845800/183634893-140a209e-9745-46d0-a7d7-2a0f381f96cc.png">


<img width="539" alt="Снимок экрана 2022-08-09 в 14 05 42" src="https://user-images.githubusercontent.com/65845800/183634929-a7610578-6c0a-4a3c-8bc3-3b8ca407aa6e.png">

<img width="575" alt="Снимок экрана 2022-08-09 в 14 05 45" src="https://user-images.githubusercontent.com/65845800/183634945-cbefadf5-e21b-4b62-837b-549689cf1416.png">

<img width="653" alt="Снимок экрана 2022-08-09 в 14 05 47" src="https://user-images.githubusercontent.com/65845800/183634963-f08a83de-cb70-4133-9733-727b18cbca9d.png">

<img width="516" alt="Снимок экрана 2022-08-09 в 14 05 52" src="https://user-images.githubusercontent.com/65845800/183634975-f0d1afc4-ee05-46ec-9ff9-6ab0801c9858.png">

<img width="386" alt="Снимок экрана 2022-08-09 в 14 06 00" src="https://user-images.githubusercontent.com/65845800/183635012-66488d8d-2f64-4902-89fd-3d35e97b2a30.png">

**Reply by chenhao-ma** [reply]
> I have the same problem on mac os, jetbrains goland 2022.2. what should i remove?
> 
> <img alt="Снимок экрана 2022-08-09 в 14 05 35" width="447" src="https://user-images.githubusercontent.com/65845800/183634893-140a209e-9745-46d0-a7d7-2a0f381f96cc.png">
> 
> <img alt="Снимок экрана 2022-08-09 в 14 05 42" width="539" src="https://user-images.githubusercontent.com/65845800/183634929-a7610578-6c0a-4a3c-8bc3-3b8ca407aa6e.png">
> 
> <img alt="Снимок экрана 2022-08-09 в 14 05 45" width="575" src="https://user-images.githubusercontent.com/65845800/183634945-cbefadf5-e21b-4b62-837b-549689cf1416.png">
> 
> <img alt="Снимок экрана 2022-08-09 в 14 05 47" width="653" src="https://user-images.githubusercontent.com/65845800/183634963-f08a83de-cb70-4133-9733-727b18cbca9d.png">
> 
> <img alt="Снимок экрана 2022-08-09 в 14 05 52" width="516" src="https://user-images.githubusercontent.com/65845800/183634975-f0d1afc4-ee05-46ec-9ff9-6ab0801c9858.png">
> 
> <img alt="Снимок экрана 2022-08-09 в 14 06 00" width="386" src="https://user-images.githubusercontent.com/65845800/183635012-66488d8d-2f64-4902-89fd-3d35e97b2a30.png">

I encountered the same issue on Mac CLion.

**Reply by Solitarystate** [reply]
I had this issue yesterday when I setup github copilot for the first time. No solutions online worked. Had to uninstall VScode on my mac and reinstall it and then sign back in and it worked.
